shtdlib.sh

```diff
@@ -60,7 +61,7 @@ os_type="$(uname)"
 
 # Determine virtualization platform in a way that ignores SIGPIPE, requires root
 if [ "${EUID}" == 0 ] && command -v virt-what &> /dev/null ; then
-    virt_platform="$(virt-what | head -1 || if [[ ${?} -eq 141 ]]; then true; else exit ${?}; fi)"
+    virt_platform=$(virt-what | head -1 || if [[ ${?} -eq 141 ]]; then true; else exit ${?}; fi)
```

Comment:
Any reason to remove the quotes?

Reply:
Disregard.. that change should not have been committed